doit/cmd_completion.py

```diff
@@ -88,7 +89,7 @@ def _generate_bash(self, opt_values, pos_args):
             pt_list_param = ''
 
         # dict with template values
-        pt_bin_name = sys.argv[0].split('/')[-1]
+        pt_bin_name = os.path.split(sys.argv[0])[1]
```

Comment:
ok for the `os.path.split` but why have you changed `-1` to `1` ?

Reply:
It doesn't really matter here. Last or second element in a [pair](https://docs.python.org/3/library/os.path.html) is the same. Do you want the -1 back?